src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -340,8 +340,11 @@ Status OnOffServer::getOnOffValue(chip::EndpointId endpoint, bool * currentOnOff
  * @param endpoint   Ver.: always
  * @param command   Ver.: always
  * @param initiatedByLevelChange   Ver.: always
+ * @param forceSend Send value of the On/Off even when it is the same as current On/Off value.
```

Comment:
Send what and where?

At startup, if some hardware state needs to be synced to the state of the On/Off cluster, the right sequence of actions is:

1. Start up the Matter stack, which initializes the On/Off cluster instance(s).
2. Read the relevant attributes from those cluster instances (which might involve the StartUpOnOff value, or might involve reading the persisted OnOff state from flash, or some combination thereof).
3. Sync the hardware state to the desired state as represented by the attributes.